versions.json

```diff
@@ -317,8 +300,22 @@
     },
     "aznfs": {
         "common": {
-            "version": "2.0.9",
-            "sha256": "36d9f34bb0554acad27f77dd2d1a7d1f8a2246a63a670d26ec2db3e680bf838f"
+            "version": "2.0.12",
+            "sha256": "5ad0f1986c4d211d2bd9626b2f246fbcd556c1fb0ca7be346adf660076c49210"
+        }
+    },
+    "rocm": {
+        "ubuntu22.04": {
+            "version": "6.2.2",
+            "url": "https://repo.radeon.com/amdgpu-install/6.2.2/ubuntu/jammy/amdgpu-install_6.2.60202-1_all.deb",
+            "sha256": "e997d1134d6e0c95dd71dd9453e78ded89016da7da6c6e8e0fdb4f204755b1e8"
+        }
+    },
+    "rccl": {
+        "ubuntu22.04": {
+            "version": "2.20.5",
```

Comment:
I know this version is from the existing file. However, as we are using ROCm 6.2.2, we should not use RCCL for ROCm 6.2.4. Please use this repo as RCCL source: https://github.com/ROCm/rccl

Please update the RCCL URL to: https://github.com/ROCm/rccl/archive/refs/tags/rocm-6.2.2.tar.gz